sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -110,13 +113,9 @@ impl BlobClient {
     /// * `options` - Optional configuration for the request.
     pub async fn download_blob(
```

Comment:
Same here as @jhendrixMSFT mentioned above. Repeating "blob" in method names on the "BlobClient" feels pretty redundant anyway.

Reply:
I'll not repeat, so probably fix up all such violations.

Reply:
Sounds good, I applied this feedback across the whole package.
`upload_blob` -> `upload`
`download_blob` -> `download`
All instances of `get_X_properties` -> `get_properties`